src/js/components/organisms/brands/brands.ts

```diff
@@ -0,0 +1,34 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { BrandsProps } from './brands.models';
+import WithRender from './brands.vue.html';
+
+@WithRender
+@Component({
+    name: 'Brands',
+})
+
+export class Brands extends Vue implements BrandsProps {
+    @Prop({ type: Object }) ctas?: BrandsProps['ctas'];
+    @Prop({ type: Object }) expand?: BrandsProps['expand'];
+    @Prop({ type: Array, required: true }) items!: BrandsProps['items'];
+    @Prop({ type: Object }) limit?: BrandsProps['limit'];
+    @Prop({ type: String }) subtitle?: BrandsProps['subtitle'];
+    @Prop({ type: String, required: true }) title!: BrandsProps['title'];
+
+    get listItems(): object {
```

Comment:
Why do something the IS should do. We should not be creating our own types. Just stick to items. In your html your are already defining that you are using a ul. then just use the <LiItem />

Reply:
resolved in discussion with @zwaardje